x-pack/plugins/cases/common/ui/types.ts

```diff
@@ -103,6 +103,15 @@ export interface ResolvedCase {
   aliasPurpose?: ResolvedSimpleSavedObject['alias_purpose'];
 }
 
+export interface Attachment {
+  id: string;
+  fileName: string | null | undefined;
+  fileType: string;
```

Comment:
This is the mime type or the extension?

Reply:
These were the initial types for the mock data so not really relevant anymore.